### DISCUSSION TITLE
Bump none-ls?

### DISCUSSION THREAD

**Discussion by Nxllpointer** [NotAShelf/nvf#452] [General]
Hi. I just noticed that none-ls is pinned to a rev from February, as some formatter I wanted to use was missing. Are there plans on updating to a new rev or should users override the input themselves if necessary?

**Comment by NotAShelf** [comment]
There are plans to *remove* null-ls as it integrates horribly with the rest of the module system. 

We use an old commit because null-ls insisted on removing """deprecated""" sources, which were actually just feature complete, to turn them into extensions. We are looking into dropping null-ls as a module, and implementing standalone formatters/linters/diagnostics in the shape of conform.nvim and nvim-lint.

**Reply by Nxllpointer** [reply]
Alright thanks
